test/index.js

```diff
@@ -923,5 +905,29 @@ describe('restify-errors node module.', function() {
                 logger.error(multiError, 'MultiError');
             });
         });
+
+        it('should serialize arbitrary fields on Error', function() {
+
+            var err1 = new Error('pull!');
+            err1.espresso = 'normale';
+
+            assert.doesNotThrow(function() {
+                logger.error(err1, 'normal error with fields');
+            });
+        });
+
+        it('should not serialize arbitrary fields on VError', function() {
+            var err1 = new verror.VError({
+                name: 'VErrorInfo',
+                info: {
+                    espresso: 'ristretto'
+                }
+            }, 'pull!');
+            err1.espresso = 'lungo';
+
+            assert.doesNotThrow(function() {
```

Comment:
These tests for the log serialize really aren't sufficient (all of them only test for throwing). I'd like to revisit them in a separate PR.